tests/test_fits.py

```diff
@@ -79,5 +80,31 @@ def test_fits_reading():
     im = imageio.imread(compressed)
     assert im.shape == (2042, 3054)
 
+@pytest.mark.skipif("astropy is None")
+def test_fits_get_reader():
```

Comment:
```suggestion
def test_fits_get_reader(tmp_path):
```

We can use the pytest fixture `tmp_path` here to get a temorary directory that will be automatically cleaned up after the test.